### PR TITLE
(WIP) Experiment to run tests with GitHub Actions

### PR DESCRIPTION
**Pull request by jmurty**
Let's see if we can get tests to run on GitHub like with Travis.

### CODE REVIEW

File: .github/workflows/run-bats-core-tests.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        git clone https://github.com/bats-core/bats-core.git /tmp/bats-core-repo
+        mkdir -p /tmp/bats-core
+        bash /tmp/bats-core-repo/install.sh /tmp/bats-core
+
+    # Need to configure git globally to avoid errors when we create and use
+    # temporary test-specific Git repos fatal: `empty ident name`
+    - name: Configure dummy Git user.[name,email] values
+      run: |
+        git config --global user.name github-actions-user
+        git config --global user.email github-actions-user@nowhere.com
```
Comment (elasticdog) [inline .github/workflows/run-bats-core-tests.yml:26]: Minor point, but I'd suggest using one of the IANA-reserved ["Special-Use Domain Names"](https://tools.ietf.org/html/rfc6761) like `example.com` or `.invalid` for this purpose, so it doesn't end up getting tied to a legit domain that someone owns.

File: .github/workflows/run-bats-core-tests.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Tests
+
+on: [push, pull_request]
```
Comment (elasticdog) [inline .github/workflows/run-bats-core-tests.yml:3]: Should we limit the push testing to just the master branch so PR testing isn't run twice?

File: .github/workflows/run-bats-core-tests.yml (hunk 1 above)
Comment (jmurty) [reply .github/workflows/run-bats-core-tests.yml]: Done

**Comment by jmurty** [comment]
@elasticdog 👍  Already [merged to master](https://github.com/elasticdog/transcrypt/commit/9fd84ff079a82aaca970a07d870561a2e1d68582) and done. I forgot to manually close this since I squash-merged away my many failed attempts

**Pull request closed without merging**